config/locales/es.yml

```diff
@@ -776,6 +776,7 @@ errors.messages.invalid_recaptcha_token: Lo sentimos, pero es posible que tu com
 errors.messages.invalid_sms_number: El número de teléfono ingresado no admite mensajes de texto. Intente la opción de llamada telefónica.
 errors.messages.invalid_voice_number: Número de teléfono no válido. Verifique haber ingresado el código de país o de área correcto.
 errors.messages.missing_field: Llene este campo.
+errors.messages.no_cert_registered: No podemos detectar un certificado en su solicitud.
```

Comment:
the non-english versions of this error are just the `blank_cert_element_req` errors. since these will not be seen by users in prod, i think this should be fine (the errors won't make much sense to users in either case) -- but i am asking the[ UX team if they feel strongly about this](https://gsa-tts.slack.com/archives/C046P4WEJDC/p1733507139634129).

Reply:
they gave me the all-clear!